src/main/resources/terraform-provider-twilio/api.mustache

```diff
@@ -103,6 +115,26 @@ func {{#vendorExtensions.x-is-create-operation}}create{{/vendorExtensions.x-is-c
     {{/vendorExtensions.x-is-delete-operation}}
 }
 
+{{#vendorExtensions.x-is-read-operation}}
+func parse{{name}}ImportId(importId string, d *schema.ResourceData) error {
+    importParts := strings.Split(importId, "/")
+    errStr := "invalid import ID (%q), expected {{#requiredParams}}{{#lambda.uppercase}}{{paramName}}{{/lambda.uppercase}}{{^-last}}/{{/-last}}{{/requiredParams}}"
+
+    if len(importParts) != {{#requiredParams}}{{#-last}}{{-index}}{{/-last}}{{/requiredParams}} {
+        return fmt.Errorf(errStr, importId)
+    }
+
+    {{#allParams}}
```

Comment:
Keeps altering between `allParams` and `requiredParams`. Could probably just use `pathParams` instead since they're all required.

Reply:
That's what I had before, except for that the pathParams don't have the snake case vendor extension.